packages/ui/packages/Carousel/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useRef } from 'react';
```

Comment:
in module it is better to use this syntax :
```js
import * as React from 'react'; 
import { useEffect, useRef } from 'react';
```

Reply:
done